public/js/GameController.js

```diff
@@ -0,0 +1,57 @@
+'use strict'
+
+class GameController {
+	
+	/** List of variables
+	* 
+	* __board 		- A GameSpace instance. Use setBoard()
```

Comment:
__board makes it sound like it's a GameBoard object.  Should be __gameSpace.  The same applies to setBoard()